backend/prompt_studio/prompt_studio_document_manager_v2/prompt_studio_document_helper.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from prompt_studio.prompt_studio_core_v2.models import CustomTool
+
+from .models import DocumentManager
+
+logger = logging.getLogger(__name__)
+
+
+class PromptStudioDocumentHelper:
+    @staticmethod
+    def create(tool_id: str, document_name: str) -> DocumentManager:
+        tool: CustomTool = CustomTool.objects.get(pk=tool_id)
+        document: DocumentManager = DocumentManager.objects.create(
```

Comment:
@muhammad-ali-e Should we not handle exceptions here?

Reply:
@harini-venkataraman What exceptions. I didn't see any  exception handler in actual v1 code.

Reply:
Will IntegrtiyError or database errors be thrown when creating the object?

Reply:
@harini-venkataraman Will that caght in global exception handler?

Reply:
@muhammad-ali-e @harini-venkataraman Let's get this checked while updating v2.
Proceeding for now.